Create Register Completed Screen

xErikPsx opened the pull request:
# Description


- New Register Completed Screen.
- There's no functionallity in "Iniciar" button until the incoming issues.

#Shout out
Thanks to @OwenRamire who helped to solve this issue.
 
  ## Screenshots 


![image](https://user-images.githubusercontent.com/26925761/96503535-01e1f500-1219-11eb-8dc8-629fef0fae69.png)

closes

OwenRamire commented on src/screens/RegisterCompleted.js:
```diff
@@ -0,0 +1,98 @@
+    textAlign: 'center',
+    flex: 0.5,
+  },
+  succesContainer: {
```
Add the "s" on the `succesContainer` (`successContainer`) property and also in the component where you use this property

OwenRamire commented on src/screens/Welcome.js:
```diff
@@ -32,7 +32,7 @@ const Welcome = ({navigation}) => {
           style={styles.button}
           onPress={() => {
-            console.log('Hello World');
+            navigation.navigate('RegisterCompleted');
```
I assume that when you navigate into the `RegisterCompleted` screen, you navigate when the  user put his contact information like this screen not when the user touches the "Iniciar sesión" button 

![image](https://user-images.githubusercontent.com/42794917/96511551-d9f88e80-1224-11eb-8f97-4575d52dbcd8.png)

OwenRamire commented on src/screens/RegisterCompleted.js:
```diff
@@ -0,0 +1,98 @@
+    flex: 1,
+    backgroundColor: Colors.white,
+  },
+
```
Delete all the white spaces that you have

OwenRamire commented on src/screens/RegisterCompleted.js:
```diff
@@ -0,0 +1,98 @@
+      </View>
+      <View style={styles.succesContainer}>
+        <View style={styles.mainText}>
+          <Text style={styles.title}>Tu cuenta se ha creado</Text>
+          <View style={styles.textContainer}>
+            <Text style={styles.paragraphs}>
+              Gracias por registrarte para protegerte y a proteger a tu
+              comunidad.
+            </Text>
+            <Text style={styles.paragraphs}>
+              Juntos podemos detener la propagación del COVID-19
+            </Text>
```
Put the text in a constant message on the file `src/utils/constants` and import them

OwenRamire commented on src/screens/RegisterCompleted.js:
```diff
@@ -20,14 +21,13 @@ const RegisterCompleted = ({navigation}) => {
       </View>
       <View style={styles.succesContainer}>
```
Change `styles.succesContainer` for `styles.successContainer`

OwenRamire committed:
Resolve rebase confict

OwenRamire committed:
Use constant messages

OwenRamire committed:
Go into Register completed screen

After the user gave us a contact information, he is going to navigate
into Register completed screens to tell him that now is regiter on the
app

OwenRamire committed:
Modify constant message

OwenRamire committed:
Clean JS console

As we use propTypes, the ResetPassConfirm screen call the
PasswordRequirements component, and this component needs props. So, when
we render this component on the ResetPassConfirm screen we don't pass
props and the JS console send us a warning because the component
requires props.

OwenRamire reviewed (dismissed):
🚀

luisarmandodv commented on src/screens/RegisterCompleted.js:
```diff
@@ -0,0 +1,102 @@
+    paddingVertical: 20,
+    flex: 2,
+  },
+
```
```suggestion
```

luisarmandodv commented on src/screens/RegisterCompleted.js:
```diff
@@ -0,0 +1,102 @@
+  mainText: {
+    flex: 2,
+  },
+
```
```suggestion
```

luisarmandodv commented on src/screens/RegisterCompleted.js:
```diff
@@ -0,0 +1,102 @@
+    height: 200,
+    resizeMode: 'contain',
+  },
+
```
```suggestion
```

luisarmandodv commented on src/screens/RegisterCompleted.js:
```diff
@@ -0,0 +1,102 @@
+          onPress={() => {
+            console.log('Register Completed');
+          }}>
+          <Text style={styles.textBtnAccept}>Iniciar</Text>
```
I see you already have a constant file for text. Could you add this text to that file?

OwenRamire committed:
Delete black spaces and add new constants
